src/main/java/controller/AccountController.java

```diff
@@ -0,0 +1,29 @@
+package controller;
+
+import command.GeneralCommand;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.Optional;
+
+public class AccountController {
```

Comment:
Servlet should be as controller, to get request and pass processing to some service/command, and then return some view (JSP page), this one is redundant and just overload the application logic.

Or there should be also controllers for all other entities (goods and user)